src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList/TaskListTabPanel.tsx

```diff
@@ -1,198 +1,199 @@
-      if (e instanceof Error) {
-        toast.error(e.message, 6000);
-      }
-    }
-  };
-
-  const createTask = async (title: string, deadline: Date, priority: TaskPriority, assignees: UserPreview[]) => {
-    try {
-      await createTaskMutate({
-        title,
-        deadline: transformDate(deadline),
-        priority,
-        status,
-        assignees: assignees.map((user) => user.userId)
-      });
-      toast.success('Task Successfully Created!');
-    } catch (e: unknown) {
-      if (e instanceof Error) {
-        toast.error(e.message, 6000);
-      }
-    }
-  };
-
-  // can the user edit this task?
-  const editTaskPermissions = (task: Task): boolean => {
-    if (!auth.user) return false;
-    return (
-      (auth.user.role === 'APP_ADMIN' ||
-        auth.user.role === 'ADMIN' ||
-        auth.user.role === 'LEADERSHIP' ||
-        auth.user.role === 'HEAD' ||
-        project.projectLead?.userId === auth.user.userId ||
-        project.projectManager?.userId === auth.user.userId ||
-        task.assignees.map((u) => u.userId).includes(auth.user.userId) ||
-        task.createdBy.userId === auth.user.userId) ??
-      false
-    );
-  };
-
-  const handleClose = () => {
-    setModalShow(false);
-    setSelectedTask(undefined);
-  };
-
-  const handleEditTask = async ({ taskId, notes, title, deadline, assignees, priority }: FormInput) => {
-    try {
-      await editTaskMutateAsync({
-        taskId,
-        notes,
-        title,
-        deadline,
-        priority
-      });
-      await editTaskAssigneesMutateAsync({
-        taskId,
-        assignees
-      });
-      toast.success('Task edited successfully!');
-    } catch (error: unknown) {
-      if (error instanceof Error) {
-        toast.error(error.message);
-      }
-    }
-    handleClose();
-  };
-
-  // Skeleton copied from https://mui.com/material-ui/react-tabs/.
-  // If they release the TabPanel component from @mui/lab to @mui/material then change the div to TabPanel.
-  return (
-    <Box flex={1}>
-      <TaskListDataGrid
-        teams={teams}
-        status={status}
-        tasks={tasks}
-        editTaskPermissions={editTaskPermissions}
-        tableRowCount={TABLE_ROW_COUNT}
-        setSelectedTask={setSelectedTask}
-        setModalShow={setModalShow}
-        addTask={addTask}
-        onAddCancel={onAddCancel}
-        createTask={createTask}
-        moveToBacklog={moveToBacklog}
-        moveToDone={moveToDone}
-        moveToInProgress={moveToInProgress}
-        deleteRow={deleteRow}
-        editTask={handleEditTask}
-        setDisabled={setDisabled}
-      />
-      {modalShow && (
-        <TaskListNotesModal
-          modalShow={modalShow}
-          onHide={handleClose}
-          onSubmit={handleEditTask}
-          task={selectedTask!}
-          teams={teams}
-          hasEditPermissions={editTaskPermissions(selectedTask!)}
-        />
-      )}
-    </Box>
-  );
-};
-
-export default TaskListTabPanel;
+/*

```

Comment:
this looks like an unintentional edit, pls revert the changes to this file, unless you were changing the imports or smthn?